test/test-PreviewProvider.js

```diff
@@ -438,6 +442,7 @@ exports.test_copy_over_correct_data_from_firefox = function*(assert) {
 exports.test_compute_image_sizes = function*(assert) {
   let mockExperimentProvider = {data: {metadataService: false}};
   gPreviewProvider = new PreviewProvider(gMockTabTracker, gMockMetadataStore, mockExperimentProvider, {initFresh: true});
+  gPreviewProvider._store = {dispatch: () => {}};
```

Comment:
oops. nvm. This is a different `gPreviewProvider` instance and you are just mocking out the `store`. :shipit: